index.d.ts

```diff
@@ -0,0 +1,22 @@
+declare module 'webext-options-sync' {
+	export interface DefineOptions {
+		defaults: Record<string, string|boolean>;
+		migrations: [(options: {disabledFeatures: string}) => void, () => void];
```

Comment:
Same as https://github.com/bfred-it/webext-options-sync/pull/15#discussion_r253711783